Sources/Sentry/SentrySDK.m

```diff
@@ -39,15 +39,20 @@
 #    import "SentryProfiler+Private.h"
 #endif // SENTRY_TARGET_PROFILING_SUPPORTED
 
+#define XCODE_PREVIEW_ENVIRONMENT_KEY @"XCODE_RUNNING_FOR_PREVIEWS"
+
 @interface SentrySDK ()
 
 @property (class) SentryHub *currentHub;
 
+#if TEST
```

Comment:
Does this need to be `TEST || TESTCI`? I'm guessing it's why tests are failing in CI.

Reply:
If im not mistaken, always running the test should have the TEST macro enabled. I will check if this is what's causing the fail, but we can add TESTCI

Reply:
```suggestion
#if TEST || TESTCI
```